lib/auth/grpcserver.go

```diff
@@ -5435,6 +5436,16 @@ func NewGRPCServer(cfg GRPCServerConfig) (*GRPCServer, error) {
 	}
 	userprovisioningpb.RegisterStaticHostUsersServiceServer(server, staticHostUserServer)
 
+	autoUpdateServiceServer, err := autoupdatev1.NewService(autoupdatev1.ServiceConfig{
+		Authorizer: cfg.Authorizer,
+		Backend:    cfg.AuthServer.Services,
+		Cache:      cfg.AuthServer.Services,
```

Comment:
this going to be replaced to `cfg.AuthServer.Cache` in next pr with cache implementation